dev/sparktestsupport/modules.py

```diff
@@ -241,7 +250,7 @@ def contains_file(self, filename):
 
 pyspark_core = Module(
     name="pyspark-core",
-    dependencies=[mllib, streaming, streaming_kafka],
+    dependencies=[mllib, streaming, streaming_kafka, streaming_flume],
```

Comment:
I think that this is a carryover from an older version of the module definitions that existed prior to splitting the `pyspark` module into multiple submodules.  I think that the right think to do is to remove the `dependencies` section from `pyspark_core` and add a `streaming_flume_assembly` dependency to `pyspark_streaming` (as I see you've already done).